### PR TITLE
Revert the change for video tag for complex object

### PR DESCRIPTION
**Pull request by VivianChu**
Update circle ci config

Fix video player for complex object. DHH-841

Update qa config.

### CODE REVIEW

**Comment by VivianChu** [comment]
@ucsdlib/developers - please review, comment, etc..  Thanks

Here are some video complex objects in qa

http://libraryqa.ucsd.edu/dc/object/bb3482439v
http://libraryqa.ucsd.edu/dc/object/bb4233646t

Here's audio complex object

http://libraryqa.ucsd.edu/dc/object/bb05129987

File: app/views/catalog/collection_search.html.erb [hunk 1]
```diff
@@ -10,7 +10,7 @@
       View current collections below. Click an image or link for a full description.
     </div>
 <%
-	@current_unit =  @current_unit.gsub!("UCSD","") if @current_unit && @current_unit.instance_of?(String) && @current_unit.include?("UCSD")
+	@current_unit =  @current_unit.gsub!("UCSD ","") if @current_unit && @current_unit.instance_of?(String) && @current_unit.include?("UCSD")
```
Comment (mcritchlow) [inline app/views/catalog/collection_search.html.erb]: why is this included in this PR? The primary change looks 👍 to me, though I've never seen that child selector syntax before. But, it looks like it works!

**Comment by VivianChu** [comment]
@mcritchlow - Ho Jung asked that extra space to be removed here https://lib-jira.ucsd.edu:8443/browse/DHH-824?focusedCommentId=47059&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-47059  I thought it's just a small change to add to this hotfix.  If not, I can remove it and add it next time.  What do you think?

**Comment by mcritchlow** [comment]
@VivianChu it's fine to get it in for the hotfix, but it's not related to this PR, so it probably should be it's own. 

@lsitu @hweng - can you please review this PR today when you have time? I'd like if we can get a hotfix in for this bug on Monday, if possible.

**Comment by hweng** [comment]
@VivianChu , @mcritchlow, It looks fine with me. I'm waiting for circleci passing the tests.

**Comment by hweng** [comment]
:+1:

**Comment by mcritchlow** [comment]
Can we squash the commits back down to one, removing the add/remove of the last 2?

**Comment by lsitu** [comment]
👍

**Comment by VivianChu** [comment]
@mcritchlow The commits have been squashed.  We have to wait for circleci to rerun the tests again.  Thanks everyone for reviewing.

**Comment by mcritchlow** [comment]
@VivianChu 👏

**Comment by VivianChu** [comment]
@ucsdlib/developers Could anybody please merge this PR?  Thanks

**Comment by VivianChu** [comment]
Thanks @lsitu for merging.

**Comment by lsitu** [comment]
@VivianChu Sure. Happy Friday!
